relay/chunkstore/chunk_writer.go

```diff
@@ -0,0 +1,103 @@
+package chunkstore
+
+import (
+	"context"
+	"fmt"
+	"github.com/Layr-Labs/eigenda/common/aws/s3"
+	"github.com/Layr-Labs/eigenda/disperser"
+	"github.com/Layr-Labs/eigenda/encoding"
+	"github.com/Layr-Labs/eigenda/encoding/rs"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/consensys/gnark-crypto/ecc/bn254"
+)
+
+// ChunkCoefficientMetadata contains metadata about how chunk coefficients are stored.
+// Required for reading chunk coefficients using ChunkReader.GetChunkCoefficients().
+type ChunkCoefficientMetadata struct {
```

Comment:
Are these the same? [encoding.FragmentInfo](https://github.com/Layr-Labs/eigenda/blob/c63dd61bf633c82ba6b30e7f71c5a17e5c452b7c/encoding/data.go#L91)

```
type FragmentInfo struct {
	TotalChunkSizeBytes uint32
	NumFragments        uint32
}
```